apps/mosaico/src/Article/Renderer.purs

```diff
@@ -0,0 +1,36 @@
+module Mosaico.Article.Renderer where
+
+import Prelude
+
+import Data.Array (cons, head, replicate)
+import Data.Foldable (fold, foldMap)
+import Data.Maybe (Maybe(..))
+import KSF.HtmlRenderer (render)
+import KSF.HtmlRenderer.Models as HtmlRenderer
+import React.Basic (JSX)
+import React.Basic.DOM as DOM
+
+renderReviewStars :: String -> JSX
+renderReviewStars content = render { content, hooks: Just [ reviewStarsHook ] }
+
+reviewStarsHook :: HtmlRenderer.HookRep
+reviewStarsHook = HtmlRenderer.replacingHook
+  { shouldProcessNode: \n ->
+     HtmlRenderer.getName n == Just "code" &&
+     HtmlRenderer.getAttrib "data-name" n == Just "stars"
```

Comment:
Are we trying to parse xml we get from drpublish here? I wouldn't like to scatter logic like this for all lettera's clients, like the mobile apps.

Reply:
There's a PR on Lettera about this now. I removed the stars generation from this end.